fr/controllers/request-response.rst

```diff
@@ -845,7 +836,7 @@ il supprime le contenu de la réponse et envoie l'en-tête `304 Not Modified`::
 
 .. _cors-headers:
 
-Définir les En-têtes de Requête d'Origine Croisée (Cross Origin Request Headers = CORS)
+Définir les En-têtes de Requête d'Origine Croisée (Cross Origin ServerRequest Headers = CORS)
 =======================================================================================
```

Comment:
Indeed and the "ServerRequest" should be "Request". The Search and Replace was a bit destructive.